core/src/test/java/org/elasticsearch/action/support/replication/TransportReplicationActionTests.java

```diff
@@ -704,7 +710,8 @@ protected void runReplicateTest(ClusterState state, IndexShardRoutingTable shard
                     // the shard the request was sent to and the shard to be failed should be the same
                     assertEquals(shardRoutingEntry.getShardRouting(), routing);
                     failures.add(shardFailedRequest);
-                    if (randomBoolean()) {
+                    int ternary = 1;
```

Comment:
That is [random](https://xkcd.com/221/) :smile:. (Left over from when I was forcing it to 1 while iterating on the test; fixed in 2912da0043a4cd1e0b80b2138c6804f692877067.)